Quantum-Control-Applications/Superconducting/Single Fixed Transmon/rb_log_xaxis.py

```diff
@@ -0,0 +1,233 @@
+            with case_(0):
+                wait(x180_len // 4, "qubit")
+            with case_(1):
+                play("x180", "qubit")
+            with case_(2):
+                play("y180", "qubit")
+            with case_(3):
+                play("y180", "qubit")
+                play("x180", "qubit")
+            with case_(4):
+                play("x90", "qubit")
+                play("y90", "qubit")
+            with case_(5):
+                play("x90", "qubit")
+                play("-y90", "qubit")
+            with case_(6):
+                play("-x90", "qubit")
+                play("y90", "qubit")
+            with case_(7):
+                play("-x90", "qubit")
+                play("-y90", "qubit")
+            with case_(8):
+                play("y90", "qubit")
+                play("x90", "qubit")
+            with case_(9):
+                play("y90", "qubit")
+                play("-x90", "qubit")
+            with case_(10):
+                play("-y90", "qubit")
+                play("x90", "qubit")
+            with case_(11):
+                play("-y90", "qubit")
+                play("-x90", "qubit")
+            with case_(12):
+                play("x90", "qubit")
+            with case_(13):
+                play("-x90", "qubit")
+            with case_(14):
+                play("y90", "qubit")
+            with case_(15):
+                play("-y90", "qubit")
+            with case_(16):
+                play("-x90", "qubit")
+                play("y90", "qubit")
+                play("x90", "qubit")
+            with case_(17):
+                play("-x90", "qubit")
+                play("-y90", "qubit")
+                play("x90", "qubit")
+            with case_(18):
+                play("x180", "qubit")
+                play("y90", "qubit")
+            with case_(19):
+                play("x180", "qubit")
+                play("-y90", "qubit")
+            with case_(20):
+                play("y180", "qubit")
+                play("x90", "qubit")
+            with case_(21):
+                play("y180", "qubit")
+                play("-x90", "qubit")
+            with case_(22):
+                play("x90", "qubit")
+                play("y90", "qubit")
+                play("x90", "qubit")
+            with case_(23):
+                play("-x90", "qubit")
+                play("y90", "qubit")
+                play("-x90", "qubit")
+
+
+with program() as rb:
+    depth = declare(int)
+    saved_gate = declare(int)
+    m = declare(int)
+    n = declare(int)
+    I = declare(fixed)
+    Q = declare(fixed)
+    state = declare(bool)
+    state_st = declare_stream()
+    d_cliffords = declare(int, value=delta_cliffords)
+    r = declare(int)
+
+    with for_(m, 0, m < num_of_sequences, m + 1):
+        sequence_list, inv_gate_list = generate_sequence()
+
+        assign(r, 0)
+
+        with for_(depth, 1, depth <= max_circuit_depth, depth + 1):
```

Comment:
This is also relevant to the original code:
Why do we have this for loop with +1 and then have the if statement inside?
Isn't it easier just to change the for loop?

This would make it much easier for both codes...